praw/exceptions.py

```diff
@@ -64,10 +64,36 @@ def __str__(self):
 
 
 class APIException(PRAWException):
-    """Old class preserved for alias purposes."""
+    """Old class preserved for alias purposes.
+
+    This class will be removed in PRAW 8.0.
+
+    .. note:: To maintain compatbility, RedditAPIException inherits from
+        APIException, therefore sharing documentation.
+    """
+
+    @classmethod
+    def _form_exception_list(
+        cls,
+        items: Union[
+            List[Union[RedditErrorItem, List[str], str]], str, RedditErrorItem
+        ],
+        *optional_args: str
+    ) -> List[Union[RedditErrorItem, List[Optional[str]]]]:
+        """Form the final list that :meth:`~._parse_exception_list` uses.
+
+        The parameters are identical to :meth:`~.__init__`.
+        """
+        if isinstance(items, str):
+            items = [[items, *optional_args]]
+        elif isinstance(items, RedditErrorItem):
+            items = [items]
+        elif isinstance(items, list) and isinstance(items[0], str):
+            items = [items] if len(items) == 3 else [items + [None]]
+        return items
 
     @staticmethod
-    def parse_exception_list(
+    def _parse_exception_list(
```

Comment:
We cannot really change this now without deprecation since it was released as a public method.

Reply:
I'll deprecate it. It should not have been public.